components/servo/Cargo.lock

```diff
@@ -2017,6 +2017,7 @@ dependencies = [
 name = "script_tests"
 version = "0.0.1"
 dependencies = [
+ "euclid 0.10.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
The command throws an error: package id specification `script` matched no package. Is it a different package name?